ast/helpers.go

```diff
@@ -0,0 +1,50 @@
+package ast
+
+import (
+	"regexp"
+	"time"
+)
+
+func IntegerLiteralFromValue(v int64) *IntegerLiteral {
```

Comment:
While these functions are all super trivial they help document that its expected to create values from literals and vice versa.